user-guides/data-farming/how-to-challengedf.md

```diff
@@ -0,0 +1,60 @@
+---
+description: Make extra dosh with building prediction models
+---
+
+# How to Challenge DF
+
+<figure><img src="../../.gitbook/assets/rewards/gif/challenge-df.gif" alt=""></figure>
+
+## Predict to earn
+
+If you're eager to begin extracting value from data but aren't sure how to get started, then [Challenge DF is here to help](../../rewards/df-challengedf.md)!
+
+By participating in the weekly data challenges of predicting the price of Ethereum, you will become familiar with the Ocean Protocol stack and with building predictions that can be sold for money.
+
+These are the steps to get involved:
+
+#### Step 1 - Navigate to the Data Farming dApp
+
+Go to https://df.oceandao.org
+
+#### Step 2 - Go to the "Challenges" page
+
+After the dApp loads, look for the menu and click on "Challenges"
+
+<figure><img src="../../.gitbook/assets/rewards/challenge-page.png" alt=""><figcaption><p>Find "Challenges" in the menu and click on it</p></figcaption></figure>
+
+#### Step 3 - Click on "Participate"
+
+After clicking on "Participate" you will be [linked to the Challenge DF README](https://github.com/oceanprotocol/predict-eth/blob/main/challenges/challenge-df.md) that outlines all details about the challenge.
+
+<figure><img src="../../.gitbook/assets/rewards/challenge-active.png" alt=""><figcaption><p>Click the Participate button</p></figcaption></figure>
+
+#### Step 4 - Use README to publish your predictions
+
+You can see the ongoing challenge right away together with some key informations such as submission deadline and available rewards.
+
+Follow the README steps to publish your model and compete in the challenge.
+
+<figure><img src="../../.gitbook/assets/rewards/challenge-df-readme.png" alt=""><figcaption><p>How to submit Challenge DF</p></figcaption></figure>
+
+#### Step 5 - Check weekly results
+
+Challenge DF weekly results become available at the end of every Data Farming round and can be viewed in the Challenges page.
+
+The top 3 competitors win prizes and are visible in the leaderboard.
```

Comment:
What are the prizes?

Reply:
The Challenge DF doc communicates the prizes so we only update them in 1 place.

Please review the top of the user-guide
```
If you're eager to begin extracting value from data but aren't sure how to get started, then [Challenge DF is here to help](../../rewards/df-challengedf.md)!
```